install

```diff
@@ -218,10 +211,6 @@ def shell_profile
   end
 end
 
-# Invalidate sudo timestamp before exiting (if it wasn't active before).
-Kernel.system "/usr/bin/sudo -n -v 2>/dev/null"
-at_exit { Kernel.system "/usr/bin/sudo", "-k" } unless $CHILD_STATUS.success?
-
```

Comment:
Instead of moving this chunk let's patch line 222.

Reply:
As `sudo` is used only when it is needed, it should be added below.

Reply:
Let's solve one issue per PR. This issue need not be address in this PR.

Reply:
This is directly related to `sudo` check below.
If a password is required, this simple check fails and does not implement `sudo -k` at exit.